homeassistant/components/intellifire/number.py

```diff
@@ -26,8 +26,8 @@ async def async_setup_entry(
     coordinator: IntellifireDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
 
     description = NumberEntityDescription(
-        key="flame_control",
-        name="Flame control",
+        key="flame_height",
```

Comment:
# ⚠️ This seems breaking!

This seems highly breaking. The unique ID is changed, but I don't see a migration in this PR.

Should this be reverted?

CC @cgarwood @jeeftor

Reply:
Good catch, completely missed that one.

Reply:
https://github.com/home-assistant/core/pull/89110 to revert this breaking change.